sample.env

```diff
@@ -0,0 +1,6 @@
+# Namecheap CPanel domain
+100DAYS_FTP_HOST=server136.web-hosting.com
+# Namecheap CPanel username
+100DAYS_FTP_USER=envinxpm
```

Comment:
I do not believe we should have this in sample

Reply:
I don't feel namecheap account name is privileged (confirmed in their support process that it's not enough to get anywhere). But I didn't feel strongly, so was down to remove. But then realized that it's part of the url path for upload. Allowing this in here means deploy can be simpler:

```
$ bundle exec rake deploy[ftp_username, ftp_password]
```

instead of requiring another param for `cpanel_account_username` which will never change.

Could we leave it, and let ftp username and password be enough?